packages/react-core/src/modules/guide/context/KnockGuideProvider.tsx

```diff
@@ -36,6 +37,7 @@ export const KnockGuideProvider: React.FC<React.PropsWithChildren<Props>> = ({
   // Default to 0 which works well for react apps as this "yields" to react for
   // one render cyle first and close the group stage.
   orderResolutionDuration = 0,
+  throttleCheckInterval,
```

Comment:
This value simply controls how often the internal `counter` gets incremented so initially I named it something like `counterIncrementInterval`, but as an external facing prop it felt too vague. I landed on just associating this prop to throttle settings, which I think is easier to reason about. 

Open to a different name if there's a better one!